scipy/stats/morestats.py

```diff
@@ -2844,6 +2844,10 @@ def wilcoxon(x, y=None, zero_method="wilcox", correction=False,
         "two-sided".
     mode : {"auto", "exact", "approx"}
         Method to calculate the p-value, see Notes. Default is "auto".
+    nan_policy : {'propagate', 'raise', 'omit'}, optional
+        Defines how to handle when input contains nan. 'propagate' just return
+        nans, 'raise' throws an error, 'omit' performs the calculations
+        ignoring nan values. Default is 'omit'.
```

Comment:
Same comment as gh-13215. Start with:
```
        Defines how to handle the occurrence of nans in `compare`.
        'propagate' returns nan, 'raise' raises an exception, 'omit'
        performs the calculations ignoring nan values. Default is
        'propagate'. Note that when the value is 'omit', nans in `scores`
        also propagate to the output, but they do not affect the z-scores
        computed for the non-nan values.
```
And strip it down, e.g.
```
        Defines how to handle the occurrence of nans in `x` and `y`.
        'propagate' returns nan, 'raise' raises an exception, 'omit'
        performs the calculations ignoring nan values. Default is
        'omit'.
```

Reply:
(I'm assuming it is OK for the default to be different in different functions? Does the guide address that?)

Reply:
I fixed it. 
> Does the guide address that?
Maybe, not. I feel "omit" should be default for `stats.wilcoxon`. But, other almost functions use `propagate` as default though.